### DISCUSSION TITLE
Create An Experiment That Requires Multiple Subjects Simultaneously

### DISCUSSION THREAD

**Discussion by angoodkind** [jspsych/jsPsych#2052] [Q&A]
This is not specific to jsPsych, so if it should be deleted, I understand. jsPsych might be a feasible solution to this, but I also figured this community would have a lot of insight.

I would like to create a job on Amazon Mechanical Turk that involves two subjects engaging in a dialogue. I already have the javascript interface coded up. How would I manage two subjects simultaneously, though? I don't want one subject to have to wait around for the second subject to join.

One workaround I was thinking about was having subjects sign up for time slots, i.e. the 5 pm slot. Then the two subjects signed up for the time slot could both join at the same time, without waiting around. Is this something I can do through Mechanical Turk?

**Comment by EceKayaMPI** [comment]
Hi, I think that is possible with the jatos server (https://www.jatos.org/Example-Group-Studies.html) but I don't know about mturk. But maybe you can get the 'batch' ideas and apply them with mturk.

**Reply by angoodkind** [reply]
This looks perfect! I want to play around with it a bit, but looks great!
